### PR TITLE
feat(1-cms): Add basic components in Homepage content CMS

### PR DESCRIPTION
**Pull request by ncuillery**
Add components:
- Header
- Spacer
- Banner

Screenshot 1|Screenshot 2
-----|----
Simulator Screenshot - iPhone 15 Pro Max - 2024-04-26 at 14 47 06|Simulator Screenshot - iPhone 15 Pro Max - 2024-04-26 at 14 47 41

### CODE REVIEW

File: lib/home/home_page_content.dart [hunk 1]
```diff
@@ -12,10 +12,28 @@ class HomePageContent extends StatelessWidget {
   @override
   Widget build(BuildContext context) {
     final widgets = sections.map((section) {
-      if (section is CarouselData) {
+      if (section is SpacerData) {
```
Comment (stephenrichter) [inline lib/home/home_page_content.dart:15]: Hmm I guess eventually we would move this to a more generic CMS display function?

File: lib/cms/models/cms_models.dart [hunk 2]
```diff
@@ -44,3 +111,27 @@ class CarouselDataItem {
...
+    // Impossible to do better with static consts (as opposed to enum)
+    switch (json) {
+      case 'sm':
+        return FlexSizes.sm;
+      case 'md':
+        return FlexSizes.md;
+      case 'lg':
+        return FlexSizes.lg;
+      default:
+        throw Exception('Unknown value for space $json');
```
Comment (stephenrichter) [inline lib/cms/models/cms_models.dart:129]: Can we default this to `FlexSizes.spacerItems`?

**Review by stephenrichter — APPROVED** [review]
Approved with a few comments

File: lib/cms/models/cms_models.dart (hunk 2 above)
Comment (ncuillery) [reply lib/cms/models/cms_models.dart:129]: Oh definitely yes 👍

File: lib/home/home_page_content.dart (hunk 1 above)
Comment (ncuillery) [reply lib/home/home_page_content.dart:15]: I haven't thought this far but yes, there is room for improvement

File: lib/cms/models/cms_models.dart (hunk 2 above)
Comment (ncuillery) [reply lib/cms/models/cms_models.dart:129]: Fixes in next PR

**Pull request merged**